static/styles/zulip.css

```diff
@@ -1752,7 +1747,12 @@ div.focused_table {
 }
 
 .messagebox-content {
-    padding: 4px 115px 1px 10px;
+    padding: 5px 109px 6px 6px;
+}
+
+.messagebox-content-wrapper {
+    border: 1px solid transparent;
```

Comment:
I kept the border always present so that the spacing is consistent.

Reply:
The normal way to do that would be with an `outline` rather than an always-present `border`. (One can still use `border-radius` to curve an `outline`.)